build.sbt

```diff
@@ -68,6 +58,4 @@ val lucumaSchemas =
     )
     .defaultAxes(VirtualAxis.jvm, VirtualAxis.scalaPartialVersion(scala2Version))
     .jvmPlatform(allVersions)
-    .jsPlatform(allVersions,
```

Comment:
Isn't this required for tests?

Reply:
Nope. Legacy setting.

Reply:
So munit doesn't need it anymore?

Reply:
Yep, as of this PR.
https://github.com/scalameta/munit/pull/376